src/librustc_mir/util/elaborate_drops.rs

```diff
@@ -323,7 +316,8 @@ impl<'l, 'b, 'tcx, D> DropCtxt<'l, 'b, 'tcx, D>
              self.elaborator.field_subpath(self.path, Field::new(i)))
         }).collect();
 
-        self.drop_ladder(fields).0
+        let (succ, unwind) = (self.succ, self.unwind); // FIXME(#6393)
```

Comment:
You can still easily track where to look from the comments in the issue (i.e. the niko’s in favour comment).